src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -43,6 +44,11 @@ public Distance getElevatorPosition() {
     return Units.Inches.of(rightMotorLeader.getPosition().getValueAsDouble());
   }
 
+  public Boolean isAtSetpoint() {
+        return (getElevatorPosition().compareTo(getElevatorPosition().minus(Constants.constElevator.DEADZONE_DISTANCE)) > 0) &&
+        getElevatorPosition().compareTo(getElevatorPosition().plus(Constants.constElevator.DEADZONE_DISTANCE)) < 0;
+  }
```

Comment:
You'll want to compare the current height to where it is trying to go to. Try passing in the set point height within the commands